src/cmd/linuxkit/run_qemu.go

```diff
@@ -398,6 +404,27 @@ func runQemuLocal(config QemuConfig) error {
 		return fmt.Errorf("Detached mode is only supported when running in a container, not locally")
 	}
 
+	if len(config.VirtiofsShares) > 0 {
+		args = append(args, "-object", "memory-backend-memfd,id=mem,size="+config.Memory+"M,share=on", "-numa", "node,memdev=mem")
+	}
+	for index, source := range config.VirtiofsShares {
+		socket := fmt.Sprintf("%s%d", "/tmp/virtiofs", index)
```

Comment:
This doesn't seem to work if I want to run multiple instances of qemu with virtiofs enabled.

Reply:
Seem it would make sense in `config.StatePath`, there is already the qemu pid in there. I'll do that, lmk if anything.